dawn/src/driver-includes/cuda_verify.hpp

```diff
@@ -64,20 +62,20 @@ __global__ void compare_dense_full_kernel(const int edge_start_idx_c, const int
     return;
   }
 
-  error[kidx * blockDim.x * gridDim.x + eidx] = compute_error<error_type>::impl(
-      fortran[kidx * dense_size_edges + eidx], dsl[kidx * dense_size_edges + eidx]);
+  error[kidx * blockDim.x * gridDim.x + eidx] =
+      compute_error<error_type>::impl(fortran[kidx * stride + eidx], dsl[kidx * stride + eidx]);
 }
 
 // Returns relative error. Prints relative and absolute error.
-inline double verify_dense_full_field(const int dense_start_idx, const int dense_end_idx,
-                                      const int dense_stride, const int k_size, const double* dsl,
-                                      const double* actual, std::string name) {
+inline double verify_field(const int start_idx, const int end_idx, const int stride,
```

Comment:
```suggestion
inline double verify_dense_field(const int start_idx, const int end_idx, const int stride,
```
cannot be used for sparse (explained above)